pgt/src/PGT/Output/TestSuite/examples/failure-on-empty-line-before-tests-without-definitions.expected.parsed

```diff
@@ -1 +1 @@
-Failed reading: expected database object definitions or test comments but found an empty line
```

Comment:
Comment: Per the parser, a test combines echo statements and results. Mentioning "test comments" in this previous error message doesn't fully convey what's required to obtain success, which in this case is a full test i.e. echo statements and test output results.